src/shared/commons/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class Constants {
```

Comment:
I am not sure but maybe we need to add Constants to case-ui-toolkit.module.ts - please run tests and demo locally. If all is good please ignore this comment.

Reply:
I have testing this locally in case management and it works all fine